lms/static/styles/components/_Dialog.scss

```diff
@@ -1,33 +1,30 @@
-.Dialog__container {
+.Dialog__background {
   display: flex;
   flex-direction: column;
+  /* Center vertically and horizontally. */
   justify-content: center;
   align-items: center;
-  position: fixed;
-  left: 0;
-  top: 0;
-  right: 0;
-  bottom: 0;
-  width: 100%;
-  height: 100%;
-}
 
-.Dialog__background {
-  background-color: black;
-  bottom: 0;
+  overflow: scroll;
+  background-color: rgba(0, 0, 0, 0.5);
+
+  /* Fill the space of the nearest positioned ancestor. */
+  position: absolute;
+  height: 100%;
+  width: 100%;
   left: 0;
-  opacity: 0.5;
-  position: fixed;
-  right: 0;
   top: 0;
 }
 
 .Dialog__content {
+  display: flex;
+  flex-direction: column;
   background-color: white;
   border-radius: 3px;
-  margin-left: auto;
-  margin-right: auto;
   padding: 20px;
+  /* Center the content horizontally in case it's width is overriden. */
+  margin-right: auto;
+  margin-left: auto;
```

Comment:
Wait, why is this needed? If the content is centered it shouldn't matter how wide it is.

Presumably it relates to this comment - https://github.com/hypothesis/lms/pull/927/files#r317419169?